frontend/public/module/k8s/types.ts

```diff
@@ -1200,3 +1200,16 @@ export type ReplicationControllerKind = {
 } & K8sResourceCommon;
 
 export type ReplicaSetKind = {} & ReplicationControllerKind;
+
+export type EndpointSliceKind = {
+  endpoints?: [
+    {
+      targetRef?: {
+        kind?: string;
+        name?: string;
+        namespace?: string;
+        uid?: string;
+      };
+    },
+  ];
+} & K8sResourceCommon;
```

Comment:
There could be multiple items in `endpoints` list right?

`endpoints is a list of unique endpoints in this slice. Each slice may include a maximum of 1000 endpoints.`

```suggestion
type EndpointSlice = {
        kind?: string;
        name?: string;
        namespace?: string;
        uid?: string;
} ;

export type EndpointSliceKind = {
  endpoints?: EndpointSlice[];
} & K8sResourceCommon;
```